chrome/github-ko.ts

```diff
@@ -239,6 +241,52 @@ const 번역목록: 번역정보[] = [
+    번역("1100H", ".col-md-9.col-12 summary", [[/Change username/, "계정명 변경"]]),
+    번역("1100I", ".col-md-9.col-12 p", [[/Looking to manage account security settings/, "계정 보안 설정을 관리하는 법을 찾고 계신가요?"], [/Export all repositories and profile metadata for/, "다음 계정에 대한 모든 저장소와 메타데이터를 내보냅니다 : "], [/Exports will be available for 7 days/, "내보내기는 7일간 유효합니다"]
+    , [/Choose how GitHub looks to you. Select a single theme, or sync with your system and automatically switch between day and night themes./, "GitHub의 테마를 선택해주세요. 하나의 테마를 선택하거나 시스템과 동기화하여 낮/밤 테마를 자동 전환하도록 할 수 있습니다"]]),
+    번역("1100J", ".col-md-9.col-12 h3", [[/Theme mode/, "테마 모드"]]),
+    번역("1100J", ".col-md-9.col-12 span", [[/GitHub theme will match your system active settings/, "깃허브 테마를 당신의 시스템 상태에 맞춰 변경되도록 합니다"]]),
+
+    // Explorer 페이지 대부분 변경완료
+    번역("1200A", ".container-lg.mx-auto.p-responsive a", [[/Explore/, "탐색"], [/Topics/, "주제"], [/Trending/, "트렌드"], [/Collections/, "컬렉션"], [/Events/, "이벤트"], [/GitHub Sponsors/, "깃허브 스폰서"], [/Get email updates/, "이메일 업데이트"]]),
+    번역("1200B", ".border-top.px-3 a", [[/starred topics/, "스타받은 토픽"]]),
+    번역("1200C", ".border-top.p-3 a", [[/starred repository/, "스타받은 저장소"]]),
+    번역("1200D", ".Box-header.d-flex a", [[/Trending repositories/, "뜨고있는 저장소들"], [/Trending developers/, "뜨고있는 개발자들"]]),
+    번역("1200E", ".col-md-8.col-lg-6.py-4 h1", [[/Here's what we found based on your interests/, "당신의 관심사를 기반으로 찾은 내용은 다음과 같습니다"]]),
+    번역("1200F", ".border.my-4 h2", [[/Based on repositories you’ve viewed/, "관심사에 맞는 저장소 추천입니다"]]),
+    번역("1200F", ".border.my-4 h2", [[/Based on topics you've starred/, "스타를 준 주제 추천입니다"]]),
+    번역("1200G", ".border-top.p-3 a", [[/See more matching repositories/, "다른 추천 저장소 보기"]]),
+    번역("1200H", ".button_to button", [[/Star/, "스타"]]),
+    번역("1200I", ".Box-footer a", [[/See more trending repositories/, "뜨고있는 저장소들 더보기"], [/See more trending developers/, "뜨고있는 개발자들 더보기"]]),
+    번역("1200J", ".col-md-8.col-lg-6.py-4 h2", [[/That's everything we found for you, for now/, "여기까지가 현재 당신을 위해 찾은 모든 것입니다."]]),
+    번역("1200J", ".col-md-8.col-lg-6.py-4 p", [[/Come back soon to see what we find next/, "우리가 다음에 무엇을 찾을 수 있는지 확인하기 위해 곧 다시 돌아오세요."]]),
+    번역("1200J", ".col-md-8.col-lg-6.py-4 span a", [[/get email updates/, "이메일 업데이트를 받아보세요"]]),
+
+// My repositories 저장소 상단, 버튼, 등 수정
+    번역("1300A", ".UnderlineNav-body a", [[/Overview/, "개요"], [/Repositories/, "저장소"], [/Projects/, "프로젝트"], [/Packages/, "패키지"]]),
+    번역("1300B", ".mb-3 button", [[/Edit profile/, "프로필 수정하기"]]),
+    번역("1300B", ".mb-3 a", [[/follower/, "팔로워"], [/following/, "팔로잉"]]),
+    번역("1300C", ".mt-4 h2", [[/Popular repositories/, "인기있는 저장소들"]]),
+    번역("1300D", "form.width-full input[name=q]", [[/Find a repository/, "저장소 찾아보기"]], 바탕값변환),
+    번역("1300D", ".d-flex.flex-wrap span", [[/Type/, "타입"], [/Language/, "언어"], [/Sort/, "분류"]]),
+    번역("1300D", ".SelectMenu-modal span", [[/Select type/, "타입 선택"], [/Select language/, "언어 선택하기"]]),
```

Comment:
```suggestion
    번역("1300D", ".SelectMenu-modal span", [[/Select type/, "타입 선택하기"], [/Select language/, "언어 선택하기"]]),
```

통일성을 위해 둘 다 '선택하기'로 하는 것이 좋을 것 같습니다!

Reply:
또는 둘다 "선택"도 좋을 것 같습니다.

Reply:
Select type과 Select Language 모두 선택으로 통일하면 깔끔할 듯 합니다!